addons/mrp_subcontracting_dropshipping/models/res_company.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    def _create_subcontracting_dropshipping_rules(self):
+        route = self.env.ref('mrp_subcontracting_dropshipping.route_subcontracting_dropshipping')
+        supplier_location = self.env.ref('stock.stock_location_suppliers')
+        vals = []
+        for company in self:
+            subcontracting_location = company.subcontracting_location_id
+            dropship_picking_type = self.env['stock.picking.type'].search([
+                ('company_id', '=', company.id),
+                ('default_location_src_id.usage', '=', 'supplier'),
+                ('default_location_dest_id.usage', '=', 'customer'),
+            ], limit=1, order='sequence')
+            vals.append({
```

Comment:
if not dropship_picking_type:
    return False

Reply:
ok, done